Examples/php/enum/Makefile

```diff
@@ -5,7 +5,7 @@ CXXSRCS    = example.cxx
 TARGET     = example
 INTERFACE  = example.i
 LIBS       =
-SWIGOPT    = -noproxy
+SWIGOPT    =
 
```

Comment:
If you rebase your branch onto current master, these `-noproxy` changes and the `Foo_IMPULSE` ones below should disappear automatically.  It's OK to do that and `git push --force` to update this PR.